lib/model/avatar_url.dart

```diff
@@ -0,0 +1,138 @@
+import '../log.dart';
+
+/// Pixel dimensions of different size choices we have (they're all
+/// square) when requesting an uploaded avatar.
+/// DEFAULT_AVATAR_SIZE in zerver/lib/upload.py.
+const int defaultUploadSizePx = 100;
```

Comment:
This text doesn't make sense when applied to just one of these definitions. It says  "different size choices we have", but this is just one choice.

Here's a revised version:
```dart
/// The size threshold above which is "medium" size for an avatar.
///
/// This is DEFAULT_AVATAR_SIZE in zerver/lib/upload.py.
const int defaultUploadSizePx = 100;
```

And then we don't actually seem to use `mediumUploadSizePx` at all, so can just drop that one.

Reply:
https://github.com/zulip/zulip-flutter/pull/534/commits/3230320ec22a46485e6d1c8c7829f3c5207aef5e